test/parallel/test-dotenv-edge-cases.js

```diff
@@ -137,6 +137,29 @@ describe('.env supports edge cases', () => {
     assert.strictEqual(child.code, 0);
   });
 
+  it('lines containing only spaces (and tabs) should not effect values defined in the next lines', async () => {
+    // Ref: https://github.com/nodejs/node/issues/56686
+    const code = `
+      process.loadEnvFile('./lines-with-only-spaces.env');
+      console.log(\`EMPTY_LINE: "\${process.env.EMPTY_LINE}"\`);
+      console.log(\`SPACES_LINE: "\${process.env.SPACES_LINE}"\`);
+      console.log(\`TABS_LINE: "\${process.env.TABS_LINE}"\`);
+      console.log(\`SPACES_TABS_LINE: "\${process.env.SPACES_TABS_LINE}"\`);
```

Comment:
Can you replace them with assert's just like in line 127

Reply:
yes, I should have done that for consistency already 😓 , although I thought it's be cleaner/clearer to have the asset in the actual js code instead of in the code string, is there any specific benefit in that? 🤔